helevents/templates/rest_framework/event_list.html

```diff
@@ -1,329 +1,411 @@
-considerable API load and long response times when too many such requests are made. Therefore,
-if you are listing the maximum number of events (100) or updating your cache with all events,
-please consider caching the keyword and location data separately to prevent unnecessary API
-slowdown and continuous repeated work. Keyword and location data seldom change and are easily
-fetched from their own endpoints separately.
-</strong></p>
-<p>Example:</p>
-<pre><code>event/?include=location,keywords
-</code></pre>
-<p><a href="?include=location,keywords" title="json">See the result</a></p>
-<h2 id="ordering">Ordering</h2>
-<p>Default ordering is descending order by <code>-last_modified_time</code>.
-You may also order results by <code>start_time</code>, <code>end_time</code>,
-<code>name</code> and <code>duration</code>. Descending order is denoted by
-adding <code>-</code> in front of the parameter, default order is ascending.
-For example:</p>
-<pre><code>event/?sort=-end_time</code></pre>
-<p><a href="?sort=-end_time" title="json">See the result</a></p>
-<h2 id="enrolment">Open enrolment</h2>
-<p>Two endpoints show the events that have connected registrations and have places either
-    at the event itself <code>enrolment_open</code> or in the waiting lists <code>
-    enrolment_open_waitlist</code>. Note that the latter query parameter when set to <code>True</code>
-    returns also the events that have open spots at the event itself. Null values are regarded
-    as unlimited number of spots at the event or in the waiting list.
-For example:</p>
-<pre><code>event/?enrolment_open_waitilist=true</code></pre>
-<p><a href="?enrolment_open_waitilist=true" title="json">See the result</a></p>
-<h3 id="event-registration">Event with registration</h3>
-<p>To find out events with a registration, use the query parameter
-<code>registration</code>.
-<p>Example:</p>
-<pre><code>event/?registration=true</code></pre>
-<p><a href="?registration=true" title="json">See the result</a></p>
-<h3 id="event-without-registration">Event without registration</h3>
-<p>To find out events without a registration, use the query parameter
-<code>registration!</code>.
-<p>Example:</p>
-<pre><code>event/?registration!=true</code></pre>
-<p><a href="?registration!=true" title="json">See the result</a></p>
+        <h3 id="local-ongoing">Ongoing local events</h3>
+        <p>Use to quickly access local (municipality level) events that are upcoming or have not ended yet. 
+            Combines the search on a number of description, name, and keyword fields. Locality is defined on 
+            the basis of MUNIGEO_MUNI value, which is set in the settings file. In the Helsinki case all the 
+            events would be retrieved that happen within Helsinki. Comes in two flavors: AND and OR. 
+            Use <code>local_ongoing_AND=lapset,musiiki</code> to search for the events with both search 
```

Comment:
```suggestion
            Use <code>local_ongoing_AND=lapset,musiikki</code> to search for the events with both search 
```

Reply:
Thanks for the suggestions and sorry about making tracking changes so difficult. Unindented html files were super difficult to read and some of them contained some extra closing tags etc. small issues. So decided to indent all the documents at the same time as updating missing query parameters to the documentation to make it more readable in the future.

Reply:
It's fine, finally someone fixes it 😁 I noticed the same a few months ago when I was adding some stuff